utils/webhook.js

```diff
@@ -0,0 +1,82 @@
+const { getResponseJSON, twilioSmsStatusTimeout } = require("./shared");
+const { processTwilioEvent, processSendGridEvent } = require("./firestore");
+const { SecretManagerServiceClient } = require("@google-cloud/secret-manager");
+const { EventWebhook, EventWebhookHeader } = require("@sendgrid/eventwebhook");
+const { getResponseJSON } = require("./shared");
+
+const sleep = (ms) => new Promise((res) => setTimeout(res, ms));
+
+/* This function will process the webhook data from Twilio */
+const receivedTwilioEvent = async (req, res) => {
```

Comment:
Can we start with a verb for this function name (eg `handleReceivedTwilioEvent` or `processReceivedTwilioEvent` )?